project/PartestUtil.scala

```diff
@@ -58,10 +58,19 @@ object PartestUtil {
     val Grep = {
       def expandGrep(x: String): Seq[String] = {
         val matchingFileContent = try {
-          val Pattern = ("(?i)" + x).r
+          val Pattern = raw"(?i)$x".r
```

Comment:
`${Pattern.quote(x)}` would also defend against regex special characters in the input.

Reply:
Tests of a broken status quo should have the form, `t555?.scala`.

Reply:
I wrote a special test file, but guess what, it turns out there was already a test with the string `hello?`.